tests/test_create_parser.py

```diff
@@ -0,0 +1,21 @@
+"""
+test_create_parser.py
+
+Tests the parser creation function.
+"""
+import pytest
+from x12.parsing import create_parser
+from x12.models import X12Delimiters
+
+
+@pytest.mark.parametrize(
```

Comment:
We will add additional parameters to this test as new transactions are supported.